ImaginaryTests-iOS/MultipleImageFetcherTests.swift

```diff
@@ -0,0 +1,58 @@
+import XCTest
+import Cache
+@testable import Imaginary
+
+fileprivate class MockDownloader: ImageDownloader {
+  var url: URL?
+  override func download(url: URL, completion: @escaping (Imaginary.Result) -> Void) {
+    self.url = url
+    DispatchQueue.global().async {
+      completion(.value(TestHelper.image()))
+    }
+  }
+}
+
+class MultipleImageFetcherTests: XCTestCase {
+  var storage: Storage!
```

Comment:
I know it's a test, but could be `private` I think.

Reply:
👍